fastly/resource_fastly_service_v1.go

```diff
@@ -1618,7 +1621,7 @@ func resourceServiceV1Update(d *schema.ResourceData, meta interface{}) error {
 		}
 
 		// update general settings
-		if d.HasChange("default_host") || d.HasChange("default_ttl") {
+		if d.HasChange("default_host") || d.HasChange("default_ttl") || (d.Get("default_ttl") == 0 && initialVersion) {
```

Comment:
Can you please add a comment above this explaining why the third conditional exists.